go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/terraform-linters/tflint
 
-go 1.23
+go 1.24.0
```

Comment:
Is the patch version needed here?

Reply:
it is needed, otherwise it would append toolchain